frontend/src/entities/group/detail/group-detail.tsx

```diff
@@ -26,17 +26,19 @@ import GroupDetailActions from './group-detail.actions';
 import Table from '../../../modules/table';
 import { groupUserColumns, visibleGroupUserColumns } from '../../user/user.columns';
 import { IGroupUser } from '../../../shared/model/groupUser.model';
-import NotificationService from '../../../shared/layout/notification/notification.service';
+import { GroupDetailUserActions } from './group-detail-user.actions';
+import { DocTitle } from '../../../shared/doc';
 
 export function GroupDetail () {
     const dispatch = useAppDispatch();
     const navigate = useNavigate();
     const {groupName} = useParams();
     const [group, setGroup] = useState<IGroup>(null);
-    const [groupUsers, setGroupUsers] = useState<IGroupUser>([]);
+    const groupUsers: IGroupUser[] = useAppSelector((state) => state.group.currentGroupUsers) || [];
```

Comment:
FWIW the project _tries_ to [write selectors](https://github.com/awslabs/mlspace/blob/develop/frontend/src/entities/configuration/configuration-reducer.ts#L150-L154) so we're not doing so much of this filtering inline and then we can define the return type so we're not having to type it ourselves. Although the example I linked to probably isn't following the best practice of prefixing it with `select...`